src/vs/workbench/contrib/webview/browser/webviewElement.ts

```diff
@@ -129,15 +129,15 @@ export class IFrameWebview extends BaseWebview<HTMLIFrameElement> implements Web
 	}
 
 	showFind(): void {
-		throw new Error('Method not implemented.');
+		// throw new Error('Method not implemented.');
```

Comment:
I think we should actually keep these exception and instead disable the find commands themselves for any webview that does not have the find widget enabled. Adding a when clause for those would prevent these methods from hit for the majority of webviews